apps/src/lab2/views/components/PredictSummary.tsx

```diff
@@ -17,18 +20,56 @@ const PredictSummary: React.FunctionComponent = () => {
     `viewAs=${ViewType.Instructor}`
   );
   const summaryUrl = document.location.pathname + SUMMARY_PATH + params;
+  const currentSectionId = useSelector(
+    (state: {teacherSections: {selectedSectionId: number}}) =>
+      state.teacherSections.selectedSectionId
+  );
+  const currentLevelId = useAppSelector(state => state.progress.currentLevelId);
+  const [responseCount, setResponseCount] = useState<number | null>(null);
+  const [numStudents, setNumStudents] = useState<number | null>(null);
+
+  useEffect(() => {
+    if (currentSectionId && currentLevelId) {
```

Comment:
I think you would still want a `catch` here since `getSectionSummary` would throw any non-`NetworkError` right? Which I guess would be rare, but it could be like a `TypeError` if something goes wrong on the client side?

Reply:
Good call, will update